From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 29 Aug 2016 22:58:49 +0200
Subject: [PATCH] build: docker: Use host's /etc/hosts

The original motivation was pointing to local mirrors via host's
"/etc/hosts", for which an alternative solution tested was
"--net=host" (see [1] for full description), but lead to wierd
loss of connectivity after a few builds.

[1] https://docs.docker.com/v1.8/articles/networking/

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/docker/runcontext | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/docker/runcontext b/build/docker/runcontext
index 07b0dc1..b0d6d37 100755
--- a/build/docker/runcontext
+++ b/build/docker/runcontext
@@ -121,6 +121,7 @@ RUN_CONTEXT_OPT="--cidfile $CID_FILE --privileged=true --rm --net=host \
     -e MIRROR_MOS_UBUNTU -e MIRROR_MOS_UBUNTU_ROOT -e MIRROR_FUEL \
     -e LATEST_TARGET_UBUNTU -e UBUNTU_ARCH -e ARMBAND_BASE -e OPNFV_GIT_SHA \
     -u $USER_ID:$GROUP_ID -w $PWD \
+    -v /etc/hosts:/etc/hosts \
     -v $GITROOT:$GITROOT -v /sys/fs/cgroup:/sys/fs/cgroup:ro $CACHEMOUNT"

 # Passing "debug" puts up an interactive bash shell
